plugin/src/makeWorklet.ts

```diff
@@ -368,6 +369,25 @@ function makeArrayFromCapturedBindings(
         currentScope = currentScope.parent;
       }
       closure.set(name, path.node);
+      isLocationSet.set(name, false);
+    },
+  });
+
+  /*
+  For reasons I don't exactly understand, the above traversal will
+  crash the whole bundle if we traversed original node instead of generated
```

Comment:
what do you mean by "the above traversal will crash the whole bundle"?

Reply:
The bundling of Reanimated Example (at least) crashes during bundling with a very vague error.

Reply:
Suggestion: replace with "the above traversal will cause the whole bundle to crash"